wireguard-go-rust-wrapper/wireguard-go/build.ps1

```diff
@@ -6,7 +6,18 @@ $ErrorActionPreference = "Stop"
 # cannot be linked with the Rust output, since Rust uses MSVC and its libraries.
 #
 # So let CLang from Visual Studio 2019 and ar.exe from TDM-GCC be our knights in shining armor.
-$env:CC="clang.exe";
+$arch = $Args[0];
+
+$env:CGO_ENABLED = "1";
+$env:GOARCH = $arch;
+
+if ($arch -eq "amd64") {
```

Comment:
I mean, what is the point of using different names here when you could just the same as in `build.rs`?

Reply:
The different value is used for `GOARCH` variable. And then for modifying `CC`.